src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -769,11 +760,7 @@ void MultiPlayerSetupGroup::force_new_dimensions(float scale,
 	log("ind. contentbox - clientbox: %d\n", max_width - clientbox.get_w());
 	log("scrollable before: %d\n", scrollable_playerbox.get_w());
 	for (auto& multiPlayerPlayerGroup : multi_player_player_groups) {
-		multiPlayerPlayerGroup->force_new_dimensions(
-		   scale,
-		   /*needs_scrolling_ ? scrollable_playerbox.get_w() - UI::Scrollbar::kSize :
-		                     scrollable_playerbox.get_w()*/
-		   max_width - clientbox.get_w(), standard_element_height);
+		multiPlayerPlayerGroup->set_desired_size(get_w() - 32 - clientbox.get_w() - UI::Scrollbar::kSize, multiPlayerPlayerGroup->get_h());
```

Comment:
thanks @Noordfrees ! I will have once again a look at the end of the week latest. What is the 32 all about?

Reply:
32 is the constant that is used for padding when the box is being created (line 666). perhaps this could be pulled out as a constexpr.